tests/UI/config.dist.js

```diff
@@ -38,12 +38,12 @@ exports.mocha = 'mocha-3.1.2';
 exports.chai = 'chai-1.9.0';
 
 /**
- * The mocha reporter to use.
+ * The multiple mocha reporters to use.
  */
-exports.reporter = "spec";
+exports.reporter = 'spec, @testomatio/reporter/lib/adapter/mocha.js';
```

Comment:
This configuration has no effect until `tests/lib/screenshot-testing/run-tests.js` is configured to use the previous `reporter: config.reporter,` instead of the fixed testomatio reporter.